script/input/11155111/common.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ilk": "ALLOCATOR-SPARK-1",
+    "usdcUnitSize": 1000,
+    "usdsUnitSize": 1000000,
+    "admin": "0x6F3066538A648b9CFad0679DF0a7e40882A23AA4"
```

Comment:
Same here. This is unused and shouldn't be set to a deployer address anyways.

Reply:
Removed